frontend/src/pages/ExecutionList.tsx

```diff
@@ -64,8 +64,8 @@ class ExecutionList extends Page<{}, ExecutionListState> {
         {
           customRenderer: this.nameCustomRenderer,
           flex: 2,
-          label: 'Pipeline/Workspace',
-          sortKey: 'pipelineName',
+          label: 'Run ID/Workspace/Pipeline',
```

Comment:
Would this column have a desirable width?

Reply:
Yes, checked with real data when I worked on this.